src/main/java/za/co/neslotech/spring/trainspringbootinsight/model/User.java

```diff
@@ -0,0 +1,54 @@
+package za.co.neslotech.spring.trainspringbootinsight.model;
+
+import jakarta.persistence.*;
+
+@Entity(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "user_id")
+    private Long userId;
```

Comment:
Keeping a value named `userId` on a `User` object seems redundant. Rather just make this `id`